static/buttons.less

```diff
@@ -31,12 +31,18 @@
   font-size: @font-size - 2px;
   height: auto;
   line-height: 1.3em;
+  &.icon:before {
+    font-size: 12px;
```

Comment:
Ya, it could. Maybe also for `sm` and `lg` buttons. And keep default buttons fixed at `16px` so they look sharp on non-retina.